src/server.h

```diff
@@ -2520,10 +2518,11 @@ void flushSlavesOutputBuffers(void);
 void disconnectSlaves(void);
 void evictClients(void);
 int listenToPort(int port, socketFds *fds);
-void pauseClients(pause_purpose purpose, mstime_t end, pause_type type);
-void unpauseClients(pause_purpose purpose);
-int areClientsPaused(void);
-int checkClientPauseTimeoutAndReturnIfPaused(void);
+void pauseServices(pause_purpose purpose, mstime_t end, uint32_t bitmask);
+void unpauseServices(pause_purpose purpose);
+uint32_t isPausedServices(uint32_t bitmask);
```

Comment:
bitmask is the type, the variable should indicate what it is.
```suggestion
uint32_t isPausedServices(uint32_t service_types);
```